http-body-util/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 #![deny(missing_debug_implementations, missing_docs, unreachable_pub)]
 #![cfg_attr(test, deny(warnings))]
+#![cfg_attr(docsrs, feature(doc(auto_cfg)))]
```

Comment:
According to the [RFC](https://rust-lang.github.io/rfcs/3631-rustdoc-cfgs-handling.html), it also needs `#![doc(auto_cfg)]`.

Reply:
Well, after run: `$ RUSTDOCFLAGS="--cfg docsrs" cargo +nightly doc --features full`

I see:
![Screenshot from 2025-04-02 18-14-51](https://github.com/user-attachments/assets/1642c32d-2911-44f8-91e4-f3239c5b2ddc)